src/scss/components/_newsDescription.scss

```diff
@@ -0,0 +1,49 @@
+.description__container{
+    padding: 60px 20px;
+}
+
+.description {
+    padding:0px;
+    @include breakpoint (lg) {
+        display: flex;
+    }
+
+    &__image {
```

Comment:
Please use `object-fit: cover;` to fix the problems in the image

Reply:
Yeah, apply it in the `<img>` tag inside this element

Reply:
Done!